docs/bug_report_template.md

```diff
@@ -0,0 +1,15 @@
+**Bug Report**
```

Comment:
For file names, either follow the convention used by GitHub issue/PR templates, or follow [our convention](https://github.com/oss-generic/process/blob/master/docs/FormatsAndConventions.md#file)?